Foundation/NSStream.swift

```diff
@@ -66,11 +69,11 @@ public class Stream: NSObject {
     }
     
     public func open() {
-        NSUnimplemented()
+        streamOpen()
```

Comment:
Please refer to the comment on #442 per requiring subclassers to implement these